src/video/wayland/SDL_waylandevents.c

```diff
@@ -1740,6 +1779,12 @@ static void seat_handle_capabilities(void *data, struct wl_seat *seat,
         wl_touch_set_user_data(input->touch, input);
         wl_touch_add_listener(input->touch, &touch_listener,
                               input);
+
+        /* Pinch gesture */
+        input->gesture_pinch = zwp_pointer_gestures_v1_get_pinch_gesture(input->pointer_gestures, input->pointer);
```

Comment:
The pinch gesture object should be destroyed when it is no longer needed.

Reply:
done